### PR TITLE
Added Player Score, MVP, Clantag, TeamID

### PR DESCRIPTION
**Pull request by derpalmer**
Just a small addition of some player fields that were missing.

### CODE REVIEW

File: entities/player.js [hunk 1]
```diff
@@ -218,6 +218,42 @@ class Player extends BaseEntity {
...
+  get mvps() {
+    let pr = this._demo.entities.getSingleton('DT_CSPlayerResource');
+    let mvps = pr.props['m_iMVPs'];
+    return mvps[Object.keys(mvps)[this.index]];
+  }
+
+  /**
+   * @returns {string} Clantag of the player
+   */
+  get clantag() {
```
Comment (saul) [inline entities/player.js]: Can we camel case this to `clanTag` please?

**Comment by derpalmer** [comment]
Sure thing ;) Done!

**Comment by saul** [comment]
Thanks for this - I'll push out a new update in a moment

**Comment by saul** [comment]
Actually - quick question about Player#teamId, how is it different to Entity#teamNumber?

**Comment by derpalmer** [comment]
Well, I missed that... But the property is different? m_iTeam and m_iTeamNum
How/when they differ I can't tell...

**Comment by saul** [comment]
The player resource entity likely duplicates some properties available on the player entities themselves. The reason that the player resource entity exists is so that clients can access the data of players that are out of the potentially visible set (PVS). For example if a client is at T spawn and another at CT spawn, it's likely that they don't have information about the other in memory as it's not networked for bandwidth/performance/anticheat reasons. However the player resource entity is always networked and so allows you to access properties of players that you don't have access to by virtue of your position relative to them. With GOTV demos, all entity data is always available, so this isn't necessary. I'll remove the property when I go to publish an update - thanks again.

**Pull request merged**